backend/src/bundles/users/user.service.ts

```diff
@@ -35,11 +36,12 @@ class UserService implements Service {
     public async create(
         payload: UserSignUpRequestDto,
     ): Promise<UserSignUpResponseDto> {
-        const { hash, salt } = cryptService.encryptSync(payload.password);
+        const { fullName, email, password } = payload;
+        const { hash, salt } = cryptService.encryptSync(password);
         const user = await this.userRepository.create(
             UserEntity.initializeNew({
-                email: payload.email,
-                fullName: payload.fullName,
+                email: email,
```

Comment:
Why can't we save email in lower case? In that way there is no need to update user entity.

Reply:
@o-nedashkivska i thought of it, but then change it, thinking in the PROD site, but you are right, thanks